apps/masterbots.ai/app/error.tsx

```
@@ -3,44 +3,42 @@ import Image from 'next/image'
-    </div>
-  )
+			<h1 className="text-3xl font-bold  mb-4">Oops! Page Not Found</h1>
+			<p className=" text-center mb-4 max-w-[480px]">{error.message}</p>
+			<Link href="/" className="flex items-center space-x-1">
+				<svg
+					width="11"
+					height="12"
+					viewBox="0 0 11 12"
+					fill="none"
+					xmlns="http://www.w3.org/2000/svg"
+				>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)